apps/api/src/main/resources/data.sql

```diff
@@ -0,0 +1,298 @@
+	       ,'Populated'                                                    AS name
+	       ,'$2a$10$ZPr0bH6kt2EnjkkRk1TEH.Mnyo/GRlfjBj/60gFuLI/BnauOx2p62' AS password
+	       ,null                                                           AS phone_number
+	       ,null                                                           AS postcode
+	       ,null                                                           AS street_address
+	       ,null                                                           AS street_address_2
+	       ,'2cd5372d-b945-48d3-83a2-82696fbb8bcd'                         AS token
+	       ,null                                                           AS updated_at
+	       ,'admin@not.real'                                               AS username)
+	UNION ALL(
+	SELECT  'ac86995f-60ae-4520-a434-c8abc98980b9'                         AS user_id
+	       ,null                                                           AS city
+	       ,null                                                           AS country
+	       ,null                                                           AS created_at
+	       ,'regular@not.real'                                             AS email
+	       ,1                                                              AS is_verified
+	       ,'Populatedtwo'                                                 AS name
+	       ,'$2a$10$ZPr0bH6kt2EnjkkRk1TEH.Mnyo/GRlfjBj/60gFuLI/BnauOx2p62' AS password
+	       ,null                                                           AS phone_number
+	       ,null                                                           AS postcode
+	       ,null                                                           AS street_address
+	       ,null                                                           AS street_address_2
+	       ,'3ceb8eda-1b03-4bf7-a85c-9d16a8ffa7ac'                         AS token
+	       ,null                                                           AS updated_at
+	       ,'regular@not.real'                                             AS username)
+) users
+WHERE NOT EXISTS (
+SELECT  *
+FROM `users` );
+-- --> End Users TABLE <-- --
+-- --> User_roles TABLE <-- --
+INSERT INTO `user_roles`
+SELECT  user_roles.*
+FROM
+( (
+	SELECT  'f12d7bcc-ac5f-48f8-8b74-055622e128f0' AS user_id
+	       ,'358417e5-813b-41b1-baa3-208603253b4c' AS role_id )
+	UNION ALL(
+	SELECT  'f12d7bcc-ac5f-48f8-8b74-055622e128f0'
+	       ,'c9bc7e4d-e1d3-42de-8ac8-2f83eae57dde' )
+	UNION ALL(
+	SELECT  'ac86995f-60ae-4520-a434-c8abc98980b9'
+	       ,'c9bc7e4d-e1d3-42de-8ac8-2f83eae57dde' )
+) user_roles
+WHERE NOT EXISTS (
+SELECT  *
+FROM `user_roles` );
+-- --> End User_roles TABLE <-- --
+-- --> Designs TABLE <-- --
+INSERT INTO `designs`
+SELECT  designs.*
+FROM
+( (
+	SELECT  'fd2532d8-52b8-4812-8bf0-baf52e6d45a6' AS design_id
+	       ,null                                   AS created_at
```

Comment:
The `createdAt` and `updatedAt` don't get automatically updated when being inserted into the database this way (normally JPA handles it)
The fields should be set to the date at which the data got populated